application-admintools-ui/src/main/resources/AdminTools/Code/AdminToolsJS.xml

```diff
@@ -271,6 +271,23 @@
       }
     }
   }
+
+  $(document).on('click', '#confirmCacheFlushModal .btn-primary', function(event) {
+    event.preventDefault();
+    const flushButton = $('#confirmCacheFlushModal .btn-primary');
+    const target = flushButton.attr('data-action')
+    $.ajax({
+      url: target,
+      type: 'POST',
+      success: function(response) {
+        $(".operationMessage").html('&lt;div class="successmessage"&gt;'+ response + '&lt;/div&gt;');
```

Comment:
I would also use `$('.health-check-section .operation-result')` (with `operation-result` instead of `operation-message`), to be sure you are getting the right element